.flake8

```diff
@@ -3,3 +3,5 @@
 builtins = _
 max-line-length = 88
 extend-ignore = E203
+exclude =
+    subiquity/system_scripts
```

Comment:
Ack. I guess this is all moot since I'm going to tear this out anyways, but I was concerned something like

```python
scripts = """\
some text
                                          really long line  # noqa
                                          another really long line  # noqa
"""
```
would be weird since it technically embeds the comment in the string, but you can just do
```python
scripts = """\
some text
                                          really long line
                                          another really long line
"""  # noqa
```
it turns out.